src/components/Registers.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Category from './Category';
+
+const RegisterPanel = ({ registers }) => {
+  const rs = registers.map((reg, i) => (
+    <li key={`{register-${i.toString()}}`}>
+      x
+      { i }
+      :
+      { reg.low_}
+    </li>
+  ));
+  return (
+    <div>
+      <Category registers={rs.slice(0, 5)} name="Pointers" />
+      <Category registers={rs.slice(5, 8).concat(rs.slice(28, 32))} name="Temporary" />
```

Comment:
should we use explanatory variables instead of magic numbers here?

Reply:
there are multiple sections in the registers where they are called "temporary", "callee-saved". maybe we can make const arrays which will hold tuples that represent a range of this type of register.

```
Pointers: [[0, 5]]
Temporary: [[8, 10], [28, 32]]

```

Reply:
if you think you will need to use these values in other parts of the app, then it could be good to create a `src/utils/constants.js` and define these constants with descriptive names, so we can reuse them.